databases/week1/db_homework_week1.sql

```diff
@@ -0,0 +1,82 @@
+--Find out how many tasks are in the task table
+SELECT
+    count (*) as "task_count"
+FROM
+    task;
+--Find out how many tasks in the task table do not have a valid due date
+SELECT
+    count (*) as "task_count"
+FROM
+    task
+WHERE
+    due_date IS NULL;
+--Find all the tasks that are marked as done
+SELECT
+    task.title AS 'DONE'
+FROM
+    task
+JOIN status ON status_id = task.status_id
+WHERE
+    status.name = 'DONE';
```

Comment:
There are two problems with this:
1. If you look at the status table, the status name for a completed task is 'Done' not 'DONE'. 
2. When you join with the status table, there is no status_id column in the status table, but id. Instead this ```status_id=task.status_id``` should be ```status.id==task.status_id```

There is a simpler way to do this: If you notice, the 'Done' status_id is 3. So, it would be enough if you queried the task table where the task.status_id = 3